pipelines/pipelines/githubapp-copyright/build-branch.yaml

```diff
@@ -72,4 +75,46 @@ spec:
       value: ""
     workspaces:
      - name: git-workspace
-       workspace: git-workspace
\ No newline at end of file
+       workspace: git-workspace
+
+  # 
+  # Assemble into a container image.
+  #
+  - name: recycle-deployment
+    taskRef:
+      name: argocd-cli
+    runAfter:
+    - docker-build-githubapp-copyright
+    params:
+    - name: server
+      value: argocd.galasa.dev
+    - name: command
+      value: 
+      - app 
+      - actions 
+      - run 
+      - $(params.appnameCli)
```

Comment:
Not clear to me where this value comes from ? What should it be ?

Reply:
This can be hard coded here, and in this case would be the app name - 'github-copyright'.

Passing the app name in as a parameter we only do to enable branch level builds when people want their own version of the maven download sites, so would have their own app in ArgoCD.